ethereumj-android/src/main/java/cedarsoftware/util/ReflectionUtils.java

```diff
@@ -0,0 +1,151 @@
+package cedarsoftware.util;
```

Comment:
This is not our package 🤔
Looks like the file was copy-pasted from some lib.
Mb better to use that lib instead?